### PR TITLE
fix: typos fixed in the repo #2108

### LINKED ISSUE #2108: Fix all typos reported by cspell
# What could be improved

**Do note that only proficient native English speakers (or proof of proficiency by published articles or similar) will be considered for this ticket, otherwise there will be too much work for the reviewers.**

All reported errors by the spell checker should either be fixed or added to the [word list](https://github.com/flame-engine/flame/blob/main/.github/cspell.json#L5).

# Why should this be improved

To rid our repository of typos.

# More information

You can either run `cspell` locally or look at what it reports in [the pipeline](https://github.com/flame-engine/flame/actions/runs/3303657189/jobs/5451857009).

### PR DESCRIPTION
**Pull request by aryabyte21**
# Description
fixed some typos

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues
Closes  #2108

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -485,7 +485,7 @@ Packages with other changes:
...
  - **REFACTOR**: Update and guarantee consistency on mocktail dev dependency version across repo ([#1701](https://github.com/flame-engine/flame/issues/1701)). ([f4a98878](https://github.com/flame-engine/flame/commit/f4a98878062dbd4fe8238a8b014e6be3e528c5d8))
  - **REFACTOR**: Add onComplete as optional parameter ([#1686](https://github.com/flame-engine/flame/issues/1686)). ([4ca65f8a](https://github.com/flame-engine/flame/commit/4ca65f8a2c330d61527e071434441f2df9deefb4))
- - **REFACTOR**: Added MultiDragListener - common API between HasDraggables and MultiTouchDragDetector ([#1668](https://github.com/flame-engine/flame/issues/1668)). ([801dbba1](https://github.com/flame-engine/flame/commit/801dbba1d8b6fd721d4e2fc752c70f97d4771198))
+ - **REFACTOR**: Added MultiDragListener - common API between HasDraggable and MultiTouchDragDetector ([#1668](https://github.com/flame-engine/flame/issues/1668)). ([801dbba1](https://github.com/flame-engine/flame/commit/801dbba1d8b6fd721d4e2fc752c70f97d4771198))
```
Comment (spydon) [inline CHANGELOG.md:488]: No fixes should be done to that changelogs, they are ignored by the spell checker (also `HasDraggables` is correct.

File: doc/flame/inputs/drag_events.md [hunk 2]
```diff
@@ -175,15 +175,15 @@ class MyComponent extends PositionComponent with DragCallbacks {
 ```
 
 
-### HasDraggablesBridge
+### HasDraggableBridge
```
Comment (spydon) [inline doc/flame/inputs/drag_events.md:178]: Faulty change, `HasDraggables` and `HasDraggablesBridge` are the names of the mixins.

**Review by spydon — COMMENTED** [review]
You shouldn't rename things in code, that will break everywhere.
All the mixin names are already correct, you can see all the mixin names here:
https://github.com/flame-engine/flame/tree/main/packages/flame/lib/src/game/mixins
and here https://github.com/flame-engine/flame/tree/main/packages/flame/lib/src/components/mixinsk

File: doc/flame/inputs/drag_events.md (hunk 2 above)
Comment (aryabyte21) [reply doc/flame/inputs/drag_events.md]: oops my bad

**Comment by spydon** [comment]
#2120 and this should be merged into one PR.

**Comment by spydon** [comment]
Any progress on this?

**Pull request closed without merging**